### PR TITLE
Useless PR

### PR DESCRIPTION
**Pull request by AniLeo**
**The perhaps useful update**
- gui: Update compatibility colors to match the new ones from the website;

**Useless janitor updates**
- rpcs3qt: Fix duplicate name warning (duplicate usage of 'verticalLayout_29');
- rpcs3qt: Removes three redundant imports;
- github: Move CONTRIBUTING to .github;
- ps3emu_api: Remove unused files that were really never used for anything;

**Useless dependency updates**
- OpenAL: Update to 1.18.2
-- No one currently uses OpenAL as Audio Render, but this doesn't break anything - nothing noticeable also benefits from basic testing, but could help in some random scenario due to bugs fixed between versions. Doesn't hurt to update;

- GL: Update header to 20180114
-- Doesn't change anything right now, just for the sake of cleaniness it's better to have the new header instead of the 2015 version. Referring to old header could eventually lead to some mistake, as there are some functions which have their parameters changed. GL_BLEND_COLOR and GL_BLEND_EQUATION were added, as well as 4.6 API.

### CODE REVIEW

**Comment by AniLeo** [comment]
Right, I forgot that was there, will do

**Comment by AniLeo** [comment]
Done

**Comment by Megamouse** [comment]
I disapprove

File: rpcs3/rpcs3qt/game_compatibility.h [hunk 1]
```diff
@@ -21,11 +21,11 @@ class game_compatibility : public QObject
...
-		{ "Intro",    { 2, "", "#f39c12", QObject::tr("Intro"),            QObject::tr("Games that display image but don't make it past the menus") } },
+		{ "Playable", { 0, "", "#1ebc61", QObject::tr("Playable"),         QObject::tr("Games that can be properly played from start to finish") } },
+		{ "Ingame",   { 1, "", "#f9b32f", QObject::tr("Ingame"),           QObject::tr("Games that either can't be finished, have serious glitches or have insufficient performance") } },
+		{ "Intro",    { 2, "", "#e08a1e", QObject::tr("Intro"),            QObject::tr("Games that display image but don't make it past the menus") } },
 		{ "Loadable", { 3, "", "#e74c3c", QObject::tr("Loadable"),         QObject::tr("Games that display a black screen with a framerate on the window's title") } },
-		{ "Nothing",  { 4, "", "#2c3e50", QObject::tr("Nothing"),          QObject::tr("Games that don't initialize properly, not loading at all and/or crashing the emulator") } },
+		{ "Nothing",  { 4, "", "#455556", QObject::tr("Nothing"),          QObject::tr("Games that don't initialize properly, not loading at all and/or crashing the emulator") } },
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/game_compatibility.h]: I don't like this color.
it should be #455555

**Pull request merged**